app/javascript/styles/mastodon/forms.scss

```diff
@@ -309,9 +309,19 @@ code {
       border-radius: 4px;
       background: url('images/void.png');
 
+      &[src$='missing.png'] {
```

Comment:
I realize this is a year old, but -- was the intention here to always hide the profile avatar image when a person has not set one yet?

Reply:
I must admit that I have no recollection of this (I can hardly remember what I had for dinner yesterday 🤪), but AFAICT this CSS is just preserving status quo? Previously the default image was inside the `if @account.header.present?` clause, so it was also hidden before, wasn't it?

Reply:
Thanks, makes sense.

I see now we need to keep some markup there for the JS hook to show preview on file presence, but don't want the missing image to be visible.

Was doing something else with responsive changes and this wasn't clear at first.